ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -402,7 +402,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node
-    optional: true
```

Comment:
undelete this

Reply:
@kikisdeliveryservice @sinnykumari feel free to adjust the PR. As far as you remember to set it a blocking afterwards, I dont have any objections

Reply:
I'm fine waiting for omer to adjust it as we don't own this test.